src/common/options.cc

```diff
@@ -1881,71 +1881,319 @@ std::vector<Option> global_options = {
 
   Option("osd_op_queue", Option::TYPE_STR, Option::LEVEL_ADVANCED)
   .set_default("wpq")
-  .set_description(""),
+  .set_enum_allowed( { "wpq", "prioritized", "mclock_opclass", "mclock_client", "debug_random" } )
+  .set_description("which operation queue algorithm to use")
+  .set_description("which operation queue algorithm to use; mclock_opclass and mclock_client are currently experimental")
```

Comment:
@ivancich does [this](http://docs.ceph.com/docs/master/dev/config/#description-and-long-description) help?